openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -594,13 +594,12 @@
 - control_key: SC-8 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        To assist with this organizational control, documentation is being
-        planned. Progress can be tracked via:
+        OpenShift uses a FIPS compliant TLS library. [1]
 
-        https://issues.redhat.com/browse/CMP-492
+        [1] https://docs.openshift.com/container-platform/4.7/installing/installing-fips.html
```

Comment:
recommended verbiage:
```yaml
        OpenShift uses a FIPS compliant TLS library [1] to assist with
        preventing unauthorized disclosures or modifications of
        information during preparation for transmission and during
        reception.

        [1] https://docs.openshift.com/container-platform/4.7/installing/installing-fips.html
```

Reply:
Any chance you could ammendment the commit and fix the verbiage? I'm out for the week